pkg/test/utils/kubernetes.go

```diff
@@ -952,23 +953,34 @@ func RunCommand(ctx context.Context, namespace string, command string, cmd harne
 		fmt.Sprintf("PATH=%s/bin/:%s", actualDir, os.Getenv("PATH")),
 	}
 
-	err = builtCmd.Run()
+	err = builtCmd.Start()
 	if err != nil {
 		if _, ok := err.(*exec.ExitError); ok && cmd.IgnoreFailure {
-			return nil
+			return nil, nil
```

Comment:
Nit: I'd rather have use always return `builtCmd`, even if it's run in foreground. This way the return value of this function isn't dependent on some options of its input. Let's have the caller decide whether they need the reference to the process or not.

Reply:
`builtCmd` is a `*exec.Cmd` which doesn't have the necessary metadata... we would have to mod `harness.Command` to have this pointer and return `harness.Command` from this func.... worth considering...  I don't get it though...  The command is run and exited... perhaps you can reason about exit codes from the calling code (which was not previously needed and we don't have a requirement for it).

Reply:
Makes sense, thanks for the explanation.